tools/PowerShell/Microsoft.WinGet.Client/src/Microsoft.WinGet.Client.psm1

```diff
@@ -0,0 +1,12 @@
+. $PSScriptRoot\Library\Get-WinGetVersion.ps1
+
+$RequiredVersion = "1.1.12653"
+
+$version = Get-WinGetVersion
+if ($version -lt $RequiredVersion) {
+	# Need to do localization
+    Write-Host "Windows Package Manager is missing. For more information on installing the Windows Package Manager. `n  Visit: https://github.com/microsoft/winget-cli#installing-the-client"
+	throw "Requires Windows Package Manager $RequiredVersion or later to be installed."
```

Comment:
I would suggest that an exception type be assigned here so that it isn't a generic `System.Exception`